### ISSUE TITLE
Show more data of *text line* column if vertial space allows

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/tables#1654]
### Is your feature request related to a problem? Please describe.

As a Nextcloud Tables user I find the available space used inefficiently with tables that have a mix of single and multi line text. Multi line text shows in, well, multiple lines. Single line text is ellipsized:

![Image](https://github.com/user-attachments/assets/9341e9d4-6de2-4b66-b4a7-71667ec73360)

### Describe the solution you'd like

Show more of the online text as wrapped text. Right now I have to click into the row editor to see more data.

### Describe alternatives you've considered

Implement the title property to allow my browser to show the full line when I hover the ellipsized text.

### Additional context

N/a

**Comment by blizzz** [comment]
I'd also argue the max line height should be limited and expendable. I believe we have it somewhere pointed out, but cannot find it now.

cc @marcoambrosini
